### ISSUE TITLE
Paste without copy to clipboard

### ISSUE THREAD

**Issue by wisemike2** [sabrogden/Ditto#670]
When click on an entry from the list, it pastes it, but at the same time it loads it to the clipboard. Is there an option to paste it without copying it to the clipboard?  Or if it could be added as a an option (feature request). Thanks.

**Comment by sabrogden** [comment]
That's how the normal pasting works from Ditto.  

You can look into Copy Buffers, options - copy buffers.  You can define a copy and paste key.  With those keys you can copy and paste outside of the current clipboard contents (behind the scenes the contents of the clipboard is saved and restored).

![image](https://github.com/sabrogden/Ditto/assets/16867884/23a3d01b-ab1e-4f13-940d-e020dffcb0c3)

So you can open Ditto select a clip and press Ctrl - num -, this will place that clip on copy buffer 1.  Ctrl - V will paste what is on the clipboard.  Press Ctrl - Num + will paste what is on copy buffer 1.  Ctrl - V will still paste the original contents of your clipboard.

**Comment by Chealer** [comment]
@wisemike2: Does Scott's reply or [that topic](https://sourceforge.net/p/ditto-cp/discussion/287511/thread/b04c90efef/) answer your question? My understanding is that it is not possible to configure Ditto so that pasting as usual does not overwrite the system's clipboard.

**Comment by wisemike2** [comment]
@Chealer  No, I couldn't find a solution so far to disable Ditto overriding the system's clipboard.  But I haven't been around this Copy Buffers suggestion, as I was talking about the default way Ditto works.

**Comment by Chealer** [comment]
@wisemike2: If what you are reporting here is the inability of Ditto to be configured so that the default paste does not overwrite the clipboard, please retitle this more specifically. I suggest:

> Default paste always overwrites Windows clipboard

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
